technische-universitat-dresden-betriebswirtschaftslehre-logistik-author-date.csl

```diff
@@ -189,15 +190,23 @@
                 <text term="in" text-case="capitalize-first" suffix=": "/>
                 <text variable="medium" suffix=", "/>
               </if>
-              <else-if type="entry-dictionary webpage post-weblog" match="any">
+              <else-if type="entry-dictionary post-weblog" match="any">
                 <text term="in" text-case="capitalize-first" suffix=": "/>
-                <text variable="container-title"/>
+                <text variable="container-title" suffix=", "/>
               </else-if>
               <else-if type="speech" match="any">
                 <text term="presented at" text-case="capitalize-first" suffix=": "/>
                 <text variable="event" suffix=", "/>
                 <text variable="event-place" suffix=", "/>
               </else-if>
+              <else-if type="webpage" match="any">
+              <choose>
+                  <if match="any" variable="container-title">
+                    <text term="in" text-case="capitalize-first" suffix=": "/>
+                    <text variable="container-title" suffix=", "/>
+                  </if>
+                </choose>
+              </else-if>
```

Comment:
It's more concise to do:

```xml
              <else-if type="webpage" match="any">
                <group delimiter=": " suffix=", ">
                  <text term="in" text-case="capitalize-first"/>
                  <text variable="container-title"/>
                </group>
              </else-if>
```

instead of 

```xml
              <else-if type="webpage" match="any">
              <choose>
                  <if match="any" variable="container-title">
                    <text term="in" text-case="capitalize-first" suffix=": "/>
                    <text variable="container-title" suffix=", "/>
                  </if>
                </choose>
              </else-if>
```

Groups are implicit conditionals. If a `<group/>` calls one or more variables, any terms in the group are automatically suppressed if the variables are empty. See http://docs.citationstyles.org/en/stable/specification.html#group for background.